modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v2/form_fields.rb

```diff
@@ -4,8 +4,8 @@ module AppealsApi
   module PdfConstruction
     module HigherLevelReview::V2
       class FormFields
-        FIRST_PAGE_ISSUES_ROW_COUNT = 7
```

Comment:
I don't think we should be assigning the value of the constant to the Structure constant. Can we just reference the constant from Structure where it's relevant?